pkg/cmd/cli/cmd/lextab.py

```diff
@@ -0,0 +1,10 @@
+_lexstateignore = {'ppline': ' \t', 'pppragma': ' \t<>.-{}();=+-*/$%@&^~!?:,0123456789', 'INITIAL': ' \t'}
+_lexstateerrorf = {'ppline': 't_ppline_error', 'pppragma': 't_pppragma_error', 'INITIAL': 't_error'}
+_lexstateeoff = {}
```

Comment:
I am not entirely sure, whenever I am building I am getting these files updated. So, I thought these are needed. I will delete them right away.